demo/web/firebase_element/app_element.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="app-element">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <div class="horizontal center-justified layout">
+      <div class="vertical-section">
+        <h3>Dinosaurs by Height</h3>
+        <template is="dom-bind">
+          <firebase-collection
+              order-by-child="height"
+              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
+              data="{{dinosaurs}}"></firebase-collection>
+          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
```

Comment:
https://github.com/PolymerElements/paper-styles/blob/master/demo/index.html is an example of how they are using the `demo-pages.html`